plugins/home/src/deprecated.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// import { createCardExtension } from './extensions';
+
+// export { createCardExtension };
+
+export * from './extensions';
```

Comment:
Ah, I think [here](https://github.com/backstage/backstage/pull/17806#discussion_r1195663539) @freben meant import from `@backstage/plugin-home-react` and re-export with "@deprecated" annotation 🤔

Reply:
@camilaibs I pushed the update! Please check if this is what was expected from this file.